ios/flutter_segment.podspec

```diff
@@ -3,21 +3,23 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'flutter_segment'
-  s.version          = '0.0.1'
-  s.summary          = 'A new flutter plugin project.'
+  s.version          = '3.1.1'
+  s.summary          = 'Segment.io plugin for Flutter'
   s.description      = <<-DESC
-A new flutter plugin project.
+Library to let Flutter apps use Segment.io
                        DESC
-  s.homepage         = 'http://example.com'
-  s.license          = { :file => '../LICENSE' }
-  s.author           = { 'Your Company' => 'email@example.com' }
-  s.source           = { :path => '.' }
+  s.homepage         = 'https://github.com/la-haus/flutter-segment'
+  s.license          = { :type => 'MIT', :file => '../LICENSE' }
+  s.author           = 'La Haus'
+  s.source           = { :git => "https://github.com/la-haus/flutter-segment.git", :tag => s.version.to_s }
+  
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
+  
   s.dependency 'Flutter'
-  s.dependency 'Analytics', '4.1.6'
-  s.dependency 'Segment-Amplitude', '3.3.2'
-  s.dependency 'segment-appsflyer-ios', '6.5.2'
+  s.dependency 'Analytics', '~> 4.1.6'
+  s.dependency 'Segment-Amplitude', '~> 3.3.2'
+  s.dependency 'segment-appsflyer-ios', '~> 6.8.0'
```

Comment:
Is there any way to make this dependencies work setting the fixed version? I mean not using the `~>`

Reply:
You mean to set it to a fixed version `6.8.0`? Yes, it is possible.
I added `~>` because it seems to me pretty much similar to what we have in Flutter with `^` symbol: https://guides.cocoapods.org/syntax/podspec.html#dependency

Reply:
Yeah, it is better to use fixed versions to have more control and have builds that are reproducible over time, also we are a bit disorganized about `.lock` files and this kind of things and we pretend to fix it in the while, so to avoid more issues with that it is better to have them fixed

Reply:
Updated the file to use fixed versions